chat-plugins/poll.js

```diff
@@ -241,17 +265,103 @@ exports.commands = {
 			if (isNaN(parsed)) return this.errorReply("To vote, specify the number of the option.");
 
 			if (!room.poll.options.has(parsed)) return this.sendReply("Option not in poll.");
+			if (room.poll.options.get(parsed).void) return this.sendReply("That option is void.");
 
 			room.poll.vote(user, parsed);
 		},
 		votehelp: ["/poll vote [number] - Votes for option [number]."],
 
+		void: function (target, room, user) {
+			if (!room.poll) return this.errorReply("There is no poll running in this room.");
+			if (!target) return this.parse('/help poll void');
+			if (!this.can('minigame', null, room)) return false;
+			let targetSplit = target.split(',');
+			for (let u in targetSplit) targetSplit[u] = targetSplit[u].trim();
+
+			if (targetSplit.length > 10) return this.errorReply("You can't void more than 10 options at a time.");
+
+			let values = [];
+			let count = 1;
+			let voided = [];
+			let invalid = [];
+
+			room.poll.options.forEach(function (obj) {
```

Comment:
You can replace `count` by using:
```javascript
room.poll.options.forEach((obj, idx) => {
```
idx is the key to the property (`obj`), since polls are setup so the keys start at 1 and increase, we can use that instead of the extra `count` variable.

Also the use of an arrow function (`=>`) is better most of the time.